### ISSUE TITLE
Step 3/10 aborted

### ISSUE THREAD

**Issue by bishav6708** [yosuken/ViPTreeGen#4]
The run gets aborted at Step 3/10. The error message goes like this:

ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
ruby: No such file or directory -- /usr/local/bin/script/01-3.cat_and_rename_split_tblastx.rb (LoadError)
rake aborted!
Command failed with status (14)

Can you help me with this?

**Comment by yosuken** [comment]
To run ViPTreeGen, "ViPTreeGen", "ViPTreeGen.rake", and "script" directory have to be in the same directory. Try to use "ViPTreeGen" command in the original directory, not at /usr/local/bin .

**Comment by bishav6708** [comment]
Thanks for the reply.
I have "ViPTreeGen" and "ViPTreeGen.rake" in /usr/local/bin directory. I ran the command from my home directory. 
I am not sure about the script directory. Is it specific for ViPTreeGen or something general?
When you mean original directory, do you mean the directory where my input files are?

**Comment by yosuken** [comment]
If you downloaded this repository to /some/path/ViPTreeGen , run /some/path/ViPTreeGen/ViPTreeGen instead of /usr/local/bin/ViPTreeGen.

**Comment by bishav6708** [comment]
Hello, Thanks for the reply. I got the program running with the following command line.
ViPTreeGen --ncpus 16 <scaffolds.fasta>~<output_directory> &
After running that, I got the distance and the similarity matrix. However, there was not no newick file. There is a file named 'all.bionj.makelog' with size 0 KB. 
The manual says that, 
--notree                   (default: off)       -- generate only similarity/distance matrix

Doesn't this mean that without any option, a newick file should be generated?

Also, I had some more questions,
1. Am I right to take the assembled scaffold file as input?
2. Does this software assigns taxonomy as well?

Thank you for your help, I rally appreciate it
